lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -476,6 +476,10 @@
     <datatype extension="protobuf3" type="galaxy.datatypes.binary:Binary" subclass="true" description="Protocol Buffers (Protobuf) is data format for serializing structured data." display_in_upload="true"/>
     <datatype extension="onnx" type="galaxy.datatypes.binary:Binary" subclass="true" description="ONNX (Open neural network exchange) is data format for storing and sharing machine learning and deep learning models." display_in_upload="true"/>
     <datatype extension="tabix" type="galaxy.datatypes.binary:Binary" subclass="true"/>
+    <datatype extension="interval_tabix" type="galaxy.datatypes.interval:IntervalTabix" mimetype="application/octet-stream" />
+    <datatype extension="juicer_medium_tabix" type="galaxy.datatypes.interval:JuicerMediumTabix" mimetype="application/octet-stream" display_in_upload="true"/>
+    <datatype extension="bed_tabix" type="galaxy.datatypes.interval:BedTabix" mimetype="application/octet-stream" display_in_upload="true"/>
+    <datatype extension="gff_tabix" type="galaxy.datatypes.interval:GffTabix" mimetype="application/octet-stream" display_in_upload="true"/>
```

Comment:
Is `.tabix` a common extension ? In that case I would suggest
```suggestion
    <datatype extension="interval.tabix" type="galaxy.datatypes.interval:IntervalTabix" mimetype="application/octet-stream" />
    <datatype extension="juicer_medium.tabix" type="galaxy.datatypes.interval:JuicerMediumTabix" mimetype="application/octet-stream" display_in_upload="true"/>
    <datatype extension="bed.tabix" type="galaxy.datatypes.interval:BedTabix" mimetype="application/octet-stream" display_in_upload="true"/>
    <datatype extension="gff.tabix" type="galaxy.datatypes.interval:GffTabix" mimetype="application/octet-stream" display_in_upload="true"/>
```
or `bed.tabix.gz` ? If we end with `.gz` tools that work with `.gz` will just work

Reply:
I think the common extension is `bgz` for bgzip (tabix only means that there is an associated tbi)... I noticed that all other extensions were with '_' that's why I put '_'...

Reply:
OK then I will change